style.css

```diff
@@ -0,0 +1,71 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  /* border: solid red; */
+  background-image:
+    url(images/ellipse_3.png),
+    url(images/ellipse_4.png),
+    url(images/ellipse_8.png),
+    url(images/ellipse_10.png),
+    url(images/ellipse_11.png),
+    url(images/Rectangle_4.png),
+    url(images/ellipse_9.png),
+    url(images/Rectangle_6.png),
+    url(images/ellipse_12.png),
+    url(images/Rectangle_5.png),
+    url(images/Rectangle_7.png),
+    url(images/ellipse_12.png),
+    url(images/ellipse_5.png),
+    url(images/ellipse_7.png);
+  background-position-x: 138%, 71%, 20%, 80%, 76%, 115%, -8%, -18%, 60%, 60%, 40%, 42%, 65%, 10%;
+  background-position-y: 20%, 95%, 87%, 78%, 82%, 5%, 15%, 15%, 15%, 100%, 15%, 17%, 45%, 96%;
+  background-repeat: no-repeat;
```

Comment:
- [ Required ] Good job implementing the `background-image`, however, please note the size of the shapes are too big compared to the Figma template provided. Please see the screenshot below and rectify this.

## Figma template
<img src="https://user-images.githubusercontent.com/16392046/212269492-f0c783cd-5397-4ccf-8286-030f74bd5c11.png" width="375px" />

## Your implementation
 
<img src="https://user-images.githubusercontent.com/16392046/212269634-db9e3897-5a5b-4177-bf91-3e740274cd10.png" width="375px" />